src/site/content/en/blog/strict-csp/index.md

```diff
@@ -153,16 +153,16 @@ To adopt a strict CSP, you need to:
 1. Decide if your application should set a nonce- or hash-based CSP.
 1. Copy the CSP from the [What is a strict Content Security
    Policy](#what-is-a-strict-content-security-policy) section and set it as a
-   response header across your application. 
+   response header across your application.
 1. Refactor HTML templates and client-side code to remove patterns that are
    incompatible with CSP.
 1. Add fallbacks to support Safari and older browsers.
 1. Deploy your CSP.
 
 You can use [Lighthouse](https://developers.google.com/web/tools/lighthouse)
-(v7.3.0 and above) **Best Practices** audit throughout this process to check
+(v7.3.0 and above with flag `--preset=experimental`) **Best Practices** audit throughout this process to check
 whether your site has a CSP, and whether it's strict enough to be effective
-against XSS. 
+against XSS.
 
 {% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/DAt65SsdOKxEHmtNFEKX.png",
```

Comment:
I have an updated screenshot for this. How can I update this image source?

Reply:
Use the uploader and copy/paste the shortcode it gives you https://web-dev-uploads.web.app/uploader